packetnetworking/distros/debian/test_bonded.py

```diff
@@ -21,8 +20,16 @@ def test_public_bonded_task_etc_network_interfaces(
     builder = bonded_network_builder(distro, version, public=True)
     tasks = builder.render()
 
+    bonding_mode = builder.network.bonding.mode
+    dns1 = builder.network.resolvers[0]
+    dns2 = builder.network.resolvers[1]
+    iface0 = builder.network.interfaces[0]
+    iface1 = builder.network.interfaces[1]
+    ipv4priv = builder.ipv4priv.first
+    ipv4pub = builder.ipv4pub.first
+    ipv6pub = builder.ipv6pub.first
     if distro == "ubuntu":
-        result = """\
+        result = f"""\
```

Comment:
Bug in a previous commit that this should be squashed into?

Reply:
nope, this commit is about using f-strings intead of `.format`